config/ingest_jorunal_example.yml

```diff
@@ -11,7 +11,7 @@ tasks:
     do:
       - if: do_ocr
         do:
-          - convert_pdf_to_tiff
+          - convert_pdf_to_tifs
```

Comment:
Ich würde vorschlagen, dass wir Formate in Tasknamen immer in der 3-Buchstaben-Form und im singular verwenden, hier also "convert_pdf_to_tif".